CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## master (unreleased)
 
+## 0.7.1 (2023-04-05)
```

Comment:
No need for this release header, as it will be added on release time. The heading here should actually be "Bugs fixed".